packages/features/auth/lib/next-auth-options.ts

```diff
@@ -788,6 +788,7 @@ export const AUTH_OPTIONS: AuthOptions = {
             username: orgId ? slugify(orgUsername) : usernameSlug(user.name),
             emailVerified: new Date(Date.now()),
             name: user.name,
+            ...(user.image && { avatarUrl: user.image }),
```

Comment:
this would save the google URL right? I wonder if we should convert this to base64 and save on our server? 🤔 @emrysal

Reply:
yeah, I think saving the URL should be fine as it's static and never expires